tokens/tests/.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
```

Comment:
IDK - this just does not feel right.
I don't like the quotes in JSON either, but this boilerplate even less.

Reply:
Done